src/api/app/models/bs_request_action.rb

```diff
@@ -37,6 +37,7 @@ class BsRequestAction < ApplicationRecord
   }
 
   before_validation :set_target_associations
+  after_save :cache_diffs
```

Comment:
This would mean for every update of every attribute we request the diff. This might not be such a good idea. Generating a diff is an expensive operation, but so can be requesting it.

Reply:
It's updated. I've moved it to `after_create`